File: app/src/main/java/com/brianmk/exposurenotes/FilmDataAdapter.java
package com.brianmk.exposurenotes;

import android.content.Context;
import android.support.annotation.NonNull;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ArrayAdapter;
import android.widget.TextView;

import java.util.List;

public class FilmDataAdapter extends ArrayAdapter<FilmData> {
    private static final String LOG_TAG = FilmDataAdapter.class.getSimpleName();

    FilmDataAdapter(Context c, List<FilmData> filmData) {
        super(c, 0, filmData);
    }

    @Override
    public @NonNull View getView(int pos, View view, @NonNull ViewGroup parent) {

        if (view == null) {
            view = LayoutInflater.from(getContext()).inflate(R.layout.activity_main,
                    parent,false);
        }

        TextView camera = view.findViewById(R.id.camera_type);


        return view;
    }


}

File: app/src/main/java/com/brianmk/exposurenotes/FilmDataAdapter.kt
package com.brianmk.exposurenotes

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ArrayAdapter
import android.widget.TextView

class FilmDataAdapter internal constructor(c: Context, filmData: List<FilmData>) : ArrayAdapter<FilmData>(c, 0, filmData) {

    override fun getView(pos: Int, view: View?, parent: ViewGroup): View {
        var view = view

        if (view == null) {
            view = LayoutInflater.from(context).inflate(R.layout.activity_main,
                    parent, false)
        }

        val camera = view!!.findViewById<TextView>(R.id.camera_type)


        return view
    }

    companion object {
        private val LOG_TAG = FilmDataAdapter::class.java.simpleName
    }


}
